### PR TITLE
Title workar

### PR DESCRIPTION
**Pull request by JelenadlS**

### CODE REVIEW

File: styles/base.css [hunk 1]
```diff
@@ -7,3 +7,18 @@
 body {
   background-color: none;
 }
+
```
Comment (ttaannjjaa) [inline styles/base.css]: according to the following code - as i have understood going with bem would mean to give also `details` and `summary` a class name to address them

File: styles/base.css [hunk 2]
```diff
@@ -7,3 +7,18 @@
...
+details > summary {
+  list-style: none;
+}
+details > summary::-webkit-details-marker {
```
Comment (ttaannjjaa) [inline styles/base.css]: wow - very cool thing - didn't see this before 👍😅

File: styles/components/app-grid.css [hunk 3]
```diff
@@ -3,8 +3,15 @@
   grid-template-rows: 48px 1fr 48px;
 }
 
-.app-grid__fixed--color {
+.app-grid__fixed-top {
```
Comment (ttaannjjaa) [inline styles/components/app-grid.css:6]: 👍 this code meets my thoughts about what's still to do and how to do best

File: styles/components/card.css [hunk 4]
```diff
@@ -9,15 +9,31 @@
...
   top: -20px;
   right: 10px;
 }
 
-.card__bookmark:active {
+.card__bookmark-btn {
+  background-color: var(--dark-orange);
+}
+
+.card__bookmark:focus {
```
Comment (ttaannjjaa) [inline styles/components/card.css:25]: Cool

**Review by ttaannjjaa — CHANGES_REQUESTED** [review]
Awesome work 😁 - i like it very much and i am learning a lot

File: styles/base.css (hunk 1 above)
Comment (JelenadlS) [reply styles/base.css]: yeah you are right, thank you 😊

File: styles/base.css (hunk 2 above)
Comment (JelenadlS) [reply styles/base.css]: haha i found it in the internet - when i now check it, nothing is changing by taking it out. so it is out again, as it seems that i fixed it somehow else 😅 🙈

**Comment by JelenadlS** [comment]
> Awesome work 😁 - i like it very much and i am learning a lot

**Review by JelenadlS — COMMENTED** [review]
Thank you Tanja for the quick check 😊

**Pull request merged**